config/e2e/rbac.yaml

```diff
@@ -86,13 +86,22 @@ rules:
       - persistentvolumeclaims
       - configmaps
       - events
+      - serviceaccounts
     verbs:
       - get
       - list
       - watch
       - delete
       - create
       - update
+  - apiGroups:
+      - "security.openshift.io"
+    resources:
+      - securitycontextconstraints
+    verbs:
+      - get
+      - update
```

Comment:
We should be able to do away with the `update` permission. Should be harmless to keep it for now.